packages/fyndiq-component-stars/src/__snapshots__/index.test.js.snap

```diff
@@ -1,48 +1,48 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`fyndiq-component-stars should be rendered with prop rating 1`] = `
+exports[`fyndiq-component-stars should be rendered with prop interactive 1`] = `
```

Comment:
`interactive` has no value, so remove the 1? Also wondering about the `NaN` in this snapshot, is that the default value of the `full` property?

Reply:
The 1 in the string is automatically added by Jest. It's to handle the cases where you do 2 calls to `expect(...).toMatchSnapshot()` in 1 test.

The `NaN` thingie, first of all, is due to how GitHub handles diff: with a better diff algorithm it shouldn't appear.

The `full` property of `<Star />` is [a number](https://github.com/fyndiq/fyndiq-ui/blob/eslint_jest/packages/fyndiq-icon-star/src/index.js#L33) and represents how full the star should be, from 0 to 1. If NaN, it will be considered as 0 (= not full). This behavior works but is a bit quirky admitedly, feel free to submit a PR that fixes that!